quirks/password-rules.json

```diff
@@ -29,6 +29,9 @@
     "autify.com": {
         "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [!\"#$%&'()*+,./:;<=>?@[^_`{|}~]];"
     },
+    "axa.de": {
+        "password-rules": "required: lower; required: upper; required: digit; allowed: [!“§$%&/()=?;:_-+*’#]; minlength: 8; maxlength: 65;"
```

Comment:
“ is the wrong one, use \\" instead

Reply:
same for '

Reply:
Thank you very much for alerting me to this. It should be fixed now.